Makefile

```diff
@@ -106,30 +108,36 @@ tags: $(GOLANG_SRCFILES) $(BPF_SRCFILES)
 	cscope -R -b -q
 
 clean-container:
-	for i in $(SUBDIRS); do $(MAKE) -C $$i clean; done
-	for i in $(SUBDIRSLIB); do $(MAKE) -C $$i clean; done
+	-for i in $(SUBDIRS); do $(MAKE) -C $$i clean; done
+	-for i in $(SUBDIRSLIB); do $(MAKE) -C $$i clean; done
 
 clean: clean-container
 	-$(MAKE) -C ./contrib/packaging/deb clean
 	-$(MAKE) -C ./contrib/packaging/rpm clean
-	-$(MAKE) -C ./contrib/packaging/docker clean
 
 install:
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	for i in $(SUBDIRS); do $(MAKE) -C $$i install; done
 	for i in $(SUBDIRSLIB); do $(MAKE) -C $$i install; done
 
-dockerfiles:
-	$(MAKE) -C ./contrib/packaging/docker dockerfiles
-
-docker-image-prod:
-	$(MAKE) -C ./contrib/packaging/docker docker-image-prod
-
-docker-image-dependencies:
-	$(MAKE) -C ./contrib/packaging/docker docker-image-dependencies
-
-docker-image-dev:
-	$(MAKE) -C ./contrib/packaging/docker docker-image-dev
+docker-image: clean
+	cat .gitignore >.dockerignore
+	echo ".*" >>.dockerignore # .git prunend out
+	echo "Documentation" >>.dockerignore # Not needed
+	echo "$(BUILD)" >BUILD_VERSION # Workaround for not having git
+	docker build -t "cilium/cilium:$(DOCKER_IMAGE_TAG)" .
+	rm BUILD_VERSION
+	echo "Push like this when ready:\ndocker push cilium/cilium:$(DOCKER_IMAGE_TAG)"
+
+docker-image-runtime:
+	cd contrib/packaging/docker && docker build -t "cilium/cilium-runtime:$(UTC_DATE)" -f Dockerfile.runtime .
+	echo "Update Dockerfile with the new tag and push like this when ready:\ndocker push cilium/cilium-runtime:$(UTC_DATE)"
```

Comment:
This description differs a little from the one under `docker-image`; is the "update dockerfile with the new tag" a separate step that requires input from the developer?